composer.json

```diff
@@ -39,6 +39,7 @@
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "1.2.*",
         "sonata-project/admin-bundle": "^2.3.3",
+        "sonata-project/translation-bundle": "1.0.0",
```

Comment:
you commented that sonata 1.0.1 has a major BC break. i was wondering when we plan to move to that and stop supporting the old version. can we do that before we release 1.3 or is it a big thing?